CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changes in sudospawner
 
+## 0.4
+
+0.4 adds better error handling:
+
+- report better errors when sudospawner fails, instead of `ValueError: substring not found` trying to parse the output
```

Comment:
...fails;  change unhelpful `ValueError: substring not found` message when parsing output fails to a more helpful message and log the unparsed data.